exporters/newrelic/src/main/java/io/opentelemetry/exporters/newrelic/NewRelicSpanExporter.java

```diff
@@ -70,4 +76,91 @@ public ResultCode export(List<Span> openTracingSpans) {
 
   @Override
   public void shutdown() {}
+
+  /**
+   * Creates a new builder instance.
+   *
+   * @return a new instance builder for this exporter.
+   */
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  /**
+   * Builder utility for this exporter. At the very minimum, you need to provide your New Relic
+   * Insert API Key for this to work.
+   *
+   * @since 0.1.0
+   */
+  public static class Builder {
+
+    private Attributes commonAttributes = new Attributes();
+    private SpanBatchSender spanBatchSender;
+    private String apiKey;
+    private boolean enableAuditLogging = false;
+
+    /**
+     * A SpanBatchSender from the New Relic Telemetry SDK. This allows you to provide your own
+     * custom-built SpanBatchSender (for instance, if you need to enable proxies, etc).
+     *
+     * @param spanBatchSender the sender to use.
+     * @return this builder's instance
+     */
+    public Builder spanBatchSender(SpanBatchSender spanBatchSender) {
+      this.spanBatchSender = spanBatchSender;
+      return this;
+    }
+
+    /**
+     * Set your New Relic Insert Key.
+     *
+     * @param apiKey your New Relic Insert Key.
+     * @return this builder's instance
+     */
+    public Builder newRelicApiKey(String apiKey) {
+      this.apiKey = apiKey;
+      return this;
+    }
+
+    /**
+     * Turn on Audit Logging for the New Relic Telemetry SDK. This will provide additional logging
+     * of the data being sent to the New Relic Trace API at DEBUG logging level.
+     *
+     * <p>WARNING: If there is sensitive data in your Traces, this will cause that data to be
+     * exposed to wherever your logs are being sent.
+     *
+     * @return this builder's instance
+     */
+    public Builder enableAuditLogging() {
+      enableAuditLogging = true;
+      return this;
+    }
+
+    /**
+     * A set of attributes that should be attached to all Spans that are sent to New Relic.
+     *
+     * @param commonAttributes the attributes to attach
+     * @return this builder's instance
+     */
+    public Builder commonAttributes(Attributes commonAttributes) {
+      this.commonAttributes = commonAttributes;
+      return this;
+    }
+
+    /**
+     * Constructs a new instance of the exporter based on the builder's values.
+     *
+     * @return a new NewRelicSpanExporter instance
+     */
+    public NewRelicSpanExporter build() {
+      if (spanBatchSender == null) {
+        SpanBatchSenderBuilder builder = SimpleSpanBatchSender.builder(apiKey);
+        if (enableAuditLogging) {
+          builder.enableAuditLogging();
+        }
+        spanBatchSender = builder.build();
+      }
+      return new NewRelicSpanExporter(new SpanBatchAdapter(commonAttributes), spanBatchSender);
+    }
+  }
```

Comment:
I'd love to have a sweet `uriOverride()` here as well for [staging] testing.

Reply:
I was trying to avoid just forwarding our entire builder over to this one. :) You can always just provide your own `SpanBatchSender` with this builder, too.

Reply:
Fair nuff.